client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -903,7 +907,7 @@ const withIsLargeCurrency = ( Component: LocalizedComponent< typeof PlanFeatures
 
 /* eslint-disable wpcalypso/redux-no-bound-selectors */
 const ConnectedPlanFeatures2023Grid = connect(
-	( state: IAppState, ownProps: PlanFeatures2023GridProps ) => {
+	( state: IAppState, ownProps: PlanFeatures2023GridType ) => {
```

Comment:
This looks a little suspect. Why change the type? Sould we also change the type of the AsyncProps passed down from `PlansFeaturesMain`?

Reply:
and I just realised we haven't merged the change yet (typed Async props). It's in https://github.com/Automattic/wp-calypso/pull/78731

Reply:
I presume that whoever merges last will have to revisit this. 🙂

Reply:
> This looks a little suspect. Why change the type?

Because `PlanFeatures2023GridType` in the intersection `PlanFeatures2023GridProps` and `PlanFeatures2023GridConnectedProps`, and AFAIK `ConnectedPlanFeatures2023Grid` has props from both types.

Also, `isGlobalStylesOnPersonal` is provided by `PlanFeatures2023GridConnectedProps`, not by `PlanFeatures2023GridProps`.

Reply:
Thanks for explaining. Never mind. False alarm - it's the exported grid's type that I'm concerned about, not anything else in this file.